### ISSUE TITLE
Adapt creating paths for ZbayChannels and OrbitDB on mobile

### ISSUE THREAD

**Issue by siepra** [ZbayApp/zbay#825]
For POC purposes we commented out creating ZbayChannels and OrbitDB dirs from storage/storage.ts and did it programatically from JAVA code. There is a need to find out if Waggle is capable of creating dirs inside app's home directory and if not - to think about better solution for preparing paths on start of Waggle.

**Comment by holmesworcester** [comment]
@siepra when you get a second, can you fill in a bit more information about what this means? Do you mean build process?

**Comment by siepra** [comment]
I've done some test and I confirm that creating paths at storage/storage.ts:63 must be commented out. Otherwise running Waggle gets stuck with 
```
2021-05-14 08:26:01.314 14670-15082/com.example.zbaypoc D/NOTICE: to get started, enter:
2021-05-14 08:26:01.315 14670-15082/com.example.zbaypoc D/NOTICE: 	jsipfs cat /ipfs/QmfGBRT6BbWJd7yUc2uYdaUZJBbnEFvTqehPFoSMQ6wgdr/readme
```

**Comment by EmiM** [comment]
https://github.com/ZbayApp/waggle/pull/40

**Comment by EmiM** [comment]
Changes merged. This will be available in waggle version >= 2.0.3.
To prevent waggle from creating dirs, one must pass `options = {createPaths: false}` to ConnectionManager constructor.

**Comment by kingalg** [comment]
Closed as per Emilia advice.
